spec/latest/core/core.jsonld

```diff
@@ -431,6 +432,16 @@
       "@id": "hydra:template",
       "@type": "rdf:Property",
       "label": "template",
+      "comment": "A templated string with placeholders. The literal's datatype indicates the template syntax; if not specified, hydra:rfc6570Template is assumed.",
+      "seeAlso": "hydra:rfc6570Template",
+      "domain": "hydra:IriTemplate",
+      "range": "rdfs:Literal",

```

Comment:
Mwah, I think that's already the job of `hydra:IriTemplate` itself. The `hydra:template` property really points to a skeleton. Syntaxes can differ, but they will be strings.